.travis.yml

```diff
@@ -24,7 +25,9 @@ gemfile:
   - gemfiles/rails5.0.gemfile
 matrix:
   fast_finish: true
+  
```

Comment:
oops, I didn't mean to add that newline. I will remove it when I squash and merge these commits.